src/background/tab-manager.ts

```diff
@@ -124,11 +133,11 @@ export default class TabManager {
             }));
     }
 
-    registerMailDisplayScript() {
-        // @ts-ignore: not a standard WebExtension API, Thunderbird only
-        chrome.messageDisplayScripts.register({
+    async registerMailDisplayScript() {
+        await (chrome as any).messageDisplayScripts.register({
             js: [
-                {file: '/inject/index.js'}
+                {file: '/inject/fallback.js'},
+                {file: '/inject/index.js'},
```

Comment:
```suggestion
                { file: '/inject/fallback.js' },
                { file: '/inject/index.js' },
```
Bracket spacing

Reply:
Running code-style over this reverts it back to normal